src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.dto.request.UserRequestDto;
+import mate.academy.spring.dto.response.UserResponseDto;
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final UserResponseMapper userResponseMapper;
```

Comment:
the same for other mappers
![Screenshot 2023-03-01 at 13 18 54](https://user-images.githubusercontent.com/60258373/222340522-d2f99482-3028-465b-8069-553dab6088b7.png)